recipes/cajun-jsonapi/all/conanfile.py

```diff
@@ -1,49 +1,89 @@
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy, load, save
+from conan.tools.scm import Version
 import os
 
+required_conan_version = ">=1.53.0"
+
+
 class CajunJsonApiConan(ConanFile):
-    name = 'cajun-jsonapi'
-    description = 'CAJUN* is a C++ API for the JSON object interchange format.'
+    name = "cajun-jsonapi"
+    description = "CAJUN* is a C++ API for the JSON object interchange format."
     topics = ("conan", "cajun", "json")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/cajun-jsonapi/cajun-jsonapi"
     license = "BSD-3-Clause"
     settings = "compiler"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+        get(
+            self,
+            **self.conan_data["sources"][self.version],
+            destination=self.source_folder,
+            strip_root=True
+        )
 
     def validate(self):
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
+        if self.info.settings.get_safe("compiler.cppstd"):
```

Comment:
```suggestion
        if self.settings.get_safe("compiler.cppstd"):
```
Keep `self.settings` when using `self.info.clear()`. Otherwise, it will return `None` always.

Reply:
I will get `ConanException: 'self.settings' access in 'validate()' method is forbidden`

Reply:
Yes, but it will be fixed on Beta-5